src/pages/ProfilePage/components/PurchaseHistory/PurchaseHistory.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import { MDBDataTableV5 } from 'mdbreact';
+import { useTranslation } from 'react-i18next';
+
+function PurchaseHistory() {
+  const { t } = useTranslation();
+
+  const [datatable] = useState({
+    columns: [
+      {
+        label: t('profilePage.orderId'),
+        field: 'orderId',
+        attributes: {
+          'aria-controls': 'DataTable',
+          'aria-label': 'orderId',
+        },
+      },
+      {
+        label: t('profilePage.books'),
+        field: 'books',
+      },
+      {
+        label: t('profilePage.status'),
+        field: 'status',
+      },
+      {
+        label: t('profilePage.originalPrice'),
+        field: 'originalPrice',
+      },
+      {
+        label: t('profilePage.totalPrice'),
+        field: 'totalPrice',
+      },
+    ],
+    rows: [
+      {
+        orderId: 'Tiger Nixon',
+        books: 'System Architect',
+        status: 'Edinburgh',
+        originalPrice: '61',
+        totalPrice: '2011/04/25',
+      },
+      {
+        orderId: 'Tiger Nixon',
+        books: 'System Architect',
+        status: 'Edinburgh',
+        originalPrice: '61',
+        totalPrice: '2011/04/25',
+      },
+      {
+        orderId: 'Tiger Nixon',
+        books: 'System Architect',
+        status: 'Edinburgh',
+        originalPrice: '61',
+        totalPrice: '2011/04/25',
+      },
+      {
+        orderId: 'Tiger Nixon',
+        books: 'System Architect',
+        status: 'Edinburgh',
+        originalPrice: '61',
+        totalPrice: '2011/04/25',
+      },
+      {
+        orderId: 'Tiger Nixon',
+        books: 'System Architect',
+        status: 'Edinburgh',
+        originalPrice: '61',
+        totalPrice: '2011/04/25',
+      },
+      {
+        orderId: 'Tiger Nixon',
+        books: 'System Architect',
+        status: 'Edinburgh',
+        originalPrice: '61',
+        totalPrice: '2011/04/25',
+      },
+      {
+        orderId: 'Tiger Nixon',
+        books: 'System Architect',
+        status: 'Edinburgh',
+        originalPrice: '61',
+        totalPrice: '2011/04/25',
+      },
+      {
+        orderId: 'Tiger Nixon',
+        books: 'System Architect',
+        status: 'Edinburgh',
+        originalPrice: '61',
+        totalPrice: '2011/04/25',
+      },
+    ],
```

Comment:
first of all number off rows is a magic number so define it as a `const NUMBER_OFF_ROWS = 8`
then you can do this
```suggestion
    rows: Array(NUMBER_OFF_ROWS ).fill({ 
        orderId: 'Tiger Nixon',
        books: 'System Architect',
        status: 'Edinburgh',
        originalPrice: '61',
        totalPrice: '2011/04/25'
        })
```